tdrs-backend/tdpservice/email/templates/request-approved.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+{% block content %}
+<!-- Body copy -->
+<p style="color: #000000;">
+    <p>Dear {{ first_name }},</p>
+
+    <p>Congratulations! Your request for the TANF Data Portal (TDP) is <b>approved</b> and access as a {{ group_permission }} from {{ stt_name }} <b>has been granted</b>.</p>
```

Comment:
noting for `OFA System Admin` awareness @jtimpe @n0remac @ttran-hub-- the following steps ensure an appropriately populated `approval` email to an STT user who requests access to TDP:

1. select group: `Data Analyst`
2. change location type to: `--------`
3. change account approval status to: `Pending`
4. click: `save and continue editing`
5. change location type to: `stt|stt`
6. change account approval status to: `Approved`
7. click: `Save`